packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -903,15 +901,29 @@ const Call = SparkPlugin.extend({
       }
     });
 
-    this.on(`change:status`, () => {
-      process.nextTick(() => this.trigger(this.status));
-    });
+    if (this.config.enableExperimentalGroupCallingSupport) {
+      this.on(`inactive`, () => this.hangup());
+      this.on(`inactive`, () => this._finish());
 
-    this.on(`change:state`, () => {
-      if (this.config.enableExperimentalGroupCallingSupport) {
+      this.on(`change:state`, () => {
         process.nextTick(() => this.trigger(this.state));
-      }
-    });
+      });
+      this.on = wrap(this.on, (fn, eventName, ...rest) => {
+        if (deprecatedCallEventNames.includes(eventName)) {
+          throw new Error(`The "${eventName}" event is no longer valid with "enableExperimentalGroupCallingSupport===true"`);
```

Comment:
May need a doc note that these events are **hard** deprecated now.

Reply:
Good call